### ISSUE TITLE
community.wallabag.org HSTS error

### ISSUE THREAD

**Issue by BeecherNetworks** [wallabag/wallabag#4698]
Wanted to ask a question however the forum mentioned in the template is not loading because of a HSTS error. Not clear why, the cert seems to be valid.

Can I ask my question in this thread or is there another platform? I don't like chat, it's too hard to follow.

**Comment by j0k3r** [comment]
We are aware of that error and we should fix it.
In the meantime I'll close your issue and you can open a new one with your question.
